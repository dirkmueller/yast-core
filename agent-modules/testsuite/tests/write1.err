[agent-modules] ModulesAgent.cc(Write):241 Obsolete interface, don't use any more!
[YCP] tests/write1.ycp:2 alias.al1 true
[agent-modules] ModulesAgent.cc(Write):241 Obsolete interface, don't use any more!
[YCP] tests/write1.ycp:4 options.op1 true
[agent-modules] ModulesAgent.cc(Write):241 Obsolete interface, don't use any more!
[YCP] tests/write1.ycp:6 pre-install.pri1 true
[agent-modules] ModulesAgent.cc(Write):241 Obsolete interface, don't use any more!
[YCP] tests/write1.ycp:7 post-install.poi1 true
[agent-modules] ModulesAgent.cc(Write):241 Obsolete interface, don't use any more!
[YCP] tests/write1.ycp:8 pre-remove.prr1 true
[agent-modules] ModulesAgent.cc(Write):241 Obsolete interface, don't use any more!
[YCP] tests/write1.ycp:9 post-remove.por1 true
