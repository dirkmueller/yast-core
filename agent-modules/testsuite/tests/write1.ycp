{
    y2warning("alias.al1 %1",Write(.alias."al1","A1"));

    y2warning("options.op1 %1",Write(.options."op1",$["OP1":"1","OP1b":""]));

    y2warning("pre-install.pri1 %1",Write(.pre-install."pri1","PRI1 PRI1b"));
    y2warning("post-install.poi1 %1",Write(.post-install."poi1","POI1 POI1b"));
    y2warning("pre-remove.prr1 %1",Write(.pre-remove."prr1","PRR1 PRR1b"));
    y2warning("post-remove.por1 %1",Write(.post-remove."por1","POR1 POR1b"));
}
