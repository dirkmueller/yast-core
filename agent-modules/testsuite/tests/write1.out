(true)
# dir test

alias al1 A1
alias al2 a2

# options
options op1 OP1=1 OP1b
options op2 o2=o2val o2a

# install
pre-install pri1 PRI1 PRI1b
pre-install pri2 prin2 prin2a prin2b

post-install poi1 POI1 POI1b
post-install poi2 poin2 poin2a poin2b

# remove
# directives
pre-remove prr1 PRR1 PRR1b
pre-remove prr2 prre2 prre2a

post-remove por1 POR1 POR1b
post-remove por2 pore2 porte2a

# keep
keep

# eof comment

